test/integration/usersFilter.test.js

```diff
@@ -46,6 +47,8 @@ describe("Filter Users", function () {
     await updateUserStatus(idleUser, generateUserStatusData("IDLE", updatedAtDate, updatedAtDate, untilDate, "CSS"));
     activeUser = await addUser(userData[8]);
     await updateUserStatus(activeUser, generateUserStatusData("ACTIVE", updatedAtDate, updatedAtDate));
+    onboardingUser = await addUser(userData[2]);
+    await updateUserStatus(onboardingUser, generateUserStatusData("ONBOARDING", updatedAtDate, updatedAtDate));
```

Comment:
I have written that line to mock the userdata(user with onboarding state) using which I can check the behaviour of API (whether it can actually filter out the onboarding users or not)